File: Python/Celsius_Fahrenheit.py
mensaje = f"""
Que deseas convertir
1: Grados Celsius a Grados Fahrenheit
2: Grados Fahrenheit a Grados Celsius
"""
print(mensaje)
mensaje = input("Ingresa 1 - 2: ")

if mensaje == "1":
    ingresaGradosCelsius = int(input("Ingresa grados Celsius: "))
    formula = (ingresaGradosCelsius * 9/5 ) + 32
    print (formula)

elif mensaje == "2":
    ingresaGradosFahrenheit = int(input("Ingresa grados Celsius: "))
    formula = (ingresaGradosFahrenheit - 32) * 5/9
    print (formula)
else:
    print("ERROR - Debes ingresar entre las dos opciones.")

File: C++/celsiusFahrenheit.cpp
#include <iostream>
using namespace std;

int main()
{
    int gradosCelsius, gradosFahrenheit, eleccion;

    cout << "1: Grados Celsius \n";
    cout << "2: Grados Fahrenheit \n";
    cout << "Elegi entre 1 - 2: ";
    cin >> eleccion;

    if (eleccion == 1)
    {
        cout << "Ingresa Grados Celsius: ";
        cin >> gradosCelsius;
        cout << ((gradosCelsius * 9 / 5) + 32);
    }

    else if (eleccion == 2)
    {
        cout << "Ingresa Grados Fahrenheit: ";
        cin >> gradosFahrenheit;
        cout << ((gradosFahrenheit - 32) * 5 / 9);
    }

    else
    {
        cout << "ERROR - Debes elegie entre las dos opciones 1 - 2.";
    }
}